packages/block-library/src/post-excerpt/edit.js

```diff
@@ -98,13 +97,11 @@ export default function PostExcerptEditor( {
 			onChange={ setExcerpt }
 			tagName="p"
 		/>
+	);
+	excerptContent = isEditable ? (
+		excerptContent
 	) : (
-		( renderedExcerpt && (
-			<Disabled>
-				<RawHTML key="html">{ renderedExcerpt }</RawHTML>
-			</Disabled>
-		) ) ||
-		__( 'No post excerpt found' )
+		<Disabled>{ excerptContent }</Disabled>
```

Comment:
I don't see a need to render RichText if we disable it anyways. Let's use the `p` tag with `dangerouslySetInnerHTML`.

We did similar changes in #38527.

https://github.com/WordPress/gutenberg/blob/cca9e4f6bd7e08261092f0dad41a7ccdfb94053d/packages/block-library/src/latest-posts/edit.js#L494-L505

Reply:
Great call, I was trying to avoid the extra `<div>` caused by the `RawHTML` component, so this should work well. I will get this updated today.

Reply:
Yesterday got away from me. Just pushed an update with this fix. Let me know what you think!